packages/experimental/src/smart-session/grantPermissions.ts

```diff
@@ -0,0 +1,70 @@
+import { ChainController, ConnectionController, OptionsController } from '@reown/appkit-core'
+import type {
+  SmartSessionGrantPermissionsRequest,
+  SmartSessionGrantPermissionsResponse
+} from './utils/TypeUtils.js'
+import {
+  assertWalletGrantPermissionsResponse,
+  extractAddress,
+  updateRequestSigner,
+  validateRequest
+} from './helper/index.js'
+import { WalletConnectCosigner } from './utils/WalletConnectCosigner.js'
```

Comment:
This is still named WalletConnect. We need to change that

Reply:
Renamed to CosignerService